packages/core/src/domain/configuration.ts

```diff
@@ -54,6 +54,7 @@ export interface UserConfiguration {
   env?: string
   version?: string
 
+  useNewIntakeDomains?: boolean
```

Comment:
since we will keep both intakes, old and new is maybe not the best naming.
Maybe something like `useAlternateIntakeDomains`?
it could worth discussing that with @hdelaby